### ISSUE TITLE
Type 0 not recognised please contant the author!

### ISSUE THREAD

**Issue by peanut2021** [matusnovak/doxybook2#43]
I met this error , and tried to use https://github.com/matusnovak/doxybook2/actions/runs/605469365 in your answer for issur #38, however it doesn't work.
Here is my xml, hope for your help, thanks.
[in.zip](https://github.com/matusnovak/doxybook2/files/6449999/in.zip)

**Comment by matusnovak** [comment]
Hi @peanut2021 Thanks for the xml files. I am currently busy but I will do my best to look at it soon.

**Comment by matusnovak** [comment]
Hi @peanut2021 

It looks like your XML files contain the `std` namespace. It's included in the `core/rt/runtime_base.h` (`runtime__base_8h.xml`) file.

The problem was that the linked `std` namespace, as `namespacestd.xml`, has references to inner STD classes that are not located alongside the rest of the XML files. It was producing a warning and it did not parse the STD namespace correctly, causing error during finalization phase.

I have pushed a fix that will handle the warning correctly and proceeds to parse the STD namespace. You will get an empty `namespacestd.md` (no inner classes) because those required XML files for STD classes do not exist, but at least it won't crash.

The fix has been released as https://github.com/matusnovak/doxybook2/releases/tag/v1.3.4
